src/App.js

```diff
@@ -1,32 +1,16 @@
-import React, { useState, useEffect } from 'react';
-import { getAllData } from './util/index';
-import Landing from './modules/Landing/components/Landing';
+import React from 'react';
+import Landing from './modules/Landing/components/Landing/Landing';
 import { Routes, Route } from "react-router-dom";
-import Login from './views/ParentViews/Login/components/Login';
+import Login from './views/ParentViews/Login/components/Login/Login';
 import './common/styles/GlobalStyles.css'
-
-const URL = 'http://localhost:8000/api/v1/';
+import ChildLogin from './views/ChildViews/components/Login/ChildLogin';
```

Comment:
You may want to think about your folder structure a little. Seems like a long nested path.

Maybe

components/ChildViews/Login

Since you can have a /ParentViews/Login too.